cmd/horcrux/cmd/threshold.go

```diff
@@ -6,27 +6,30 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/strangelove-ventures/horcrux/pkg/pcosigner"
+
+	"github.com/strangelove-ventures/horcrux/pkg/node"
+
 	cometlog "github.com/cometbft/cometbft/libs/log"
 	cometservice "github.com/cometbft/cometbft/libs/service"
-	"github.com/strangelove-ventures/horcrux/signer"
 )
 
 const maxWaitForSameBlockAttempts = 3
 
 func NewThresholdValidator(
 	logger cometlog.Logger,
-) ([]cometservice.Service, *signer.ThresholdValidator, error) {
+) ([]cometservice.Service, *node.ThresholdValidator, error) {
 	if err := config.Config.ValidateThresholdModeConfig(); err != nil {
 		return nil, nil, err
 	}
 
 	thresholdCfg := config.Config.ThresholdModeConfig
-
-	remoteCosigners := make([]signer.Cosigner, 0, len(thresholdCfg.Cosigners)-1)
+	// NOTE: Shouldnt this be a list of concrete type instead of interface type?
```

Comment:
no, because `NewThresholdValidator` takes the interface type. This lets us use local cosigners for the remote cosigners in the tests.

Reply:
Correct! :)